.github/workflows/long-running-azure.yaml

```
@@ -515,6 +515,7 @@ jobs:
             --yes --verbose
       - name: Clean up cluster
         if: always()
+        timeout-minutes: 60
```

Added timeout to clean up step.

---

What's the goal of the timeout? That we'd be notified if we hit it?

---

Default is 6 hours, so fail faster?

---

Exactly. To fail faster.